vspec/loggingconfig.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# (c) 2022 BMW Group
```

Comment:
As this is a totally new file, not copied from any existing one you could if you like remove BMW/BoSch/JLR here. If you like you could instead add your own organization.

(The practical approach so far concerning copyright have been that if you do significant contributions to a file then you could add your organization to copyright, if you like)